admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/utils/ZoneUtils.java

```diff
@@ -250,8 +253,8 @@ private void searchZones(List<Zone> zones, List<Zone> zoneIds, ZoneUser zu) {
 			List<Zone> langSpecificZones = null;
 			ZoneUser zu=zoneUserRepository.findZoneByUserIdActiveAndNonDeleted(((AuthUserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getUserId());
 
-		if (langCode.equals("all")) {
-			String lang=LanguageUtils.getLanguage();
+		if (langCode.equals("all") || langCode==null) {
```

Comment:
this leads to NPE, put null check before "all" check

Reply:
taken care